micropython/build.sh

```diff
@@ -21,24 +22,51 @@ mkdir -p "$PREFIX_UPYTH_BUILD" "$PREFIX_UPYTH_MARKERS"
 
 
 #
-# Apply patches
+# Apply patches and copy files
 #
-# FIXME: prefix patch files with numbers to ensure correct order
-for patchfile in "$PREFIX_UPYTH_PATCHSET"/*.patch; do
+for patchfile in "$PREFIX_UPYTH_CONFIG"/patches/*.patch; do
 	if [ ! -f "$PREFIX_UPYTH_MARKERS/$(basename "$patchfile").applied" ]; then
 		echo "applying patch: $patchfile"
-		patch -d "$PREFIX_UPYTH_SRC" -p1 < "$patchfile"   # FIXME: should be -p1
+		patch -d "$PREFIX_UPYTH_SRC" -p1 < "$patchfile"
 		touch "$PREFIX_UPYTH_MARKERS/$(basename "$patchfile").applied"
 	fi
 done
+cp -a "${PREFIX_UPYTH_CONFIG}/files/001_mpconfigport.mk" "${PREFIX_UPYTH_SRC}/ports/unix/mpconfigport.mk" && echo "Copied mpconfigport.mk!"
+
+
+#
+# Micropython internal use stack/heap size (not actual application stack/heap)
+# Values are to be overwritten in _targets/build.project.*
+#
+: "${UPYTH_STACKSZ=4096}"
+: "${UPYTH_HEAPSZ=16384}"
+
+
+#
+# Architecture specific flags/values set
+#
+if [ "${TARGET_FAMILY}" = "armv7m7" ]; then
+	STRIPEXP="--strip-unneeded"
+elif [ "${TARGET_FAMILY}" = "ia32" ]; then
+	STRIPEXP="--strip-all"
+else
+	b_die "ERROR! - No architecture for micropython available"
```

Comment:
not sure we want to explicitly forbid compilation on other platforms (I've run micropython successfully on `imx6ull` even if it wasn't tested by You :p).

I think we may add a warning message that it's not officially supported or something like that but allow compilation anyway

Reply:
Added warning message with 5 second wait to give user a chance to see it if it happens